### DISCUSSION TITLE
Unable to override defaults

### DISCUSSION THREAD

**Discussion by ephraimduncan** [nextauthjs/next-auth#4574] [Help]
### Question 💬

I am trying to override the default scope of the Twitter Provider. I have done the override in the config file but the token object is returning the default scope. 

```js
TwitterProvider({
    clientId: process.env.TWITTER_ID,
    clientSecret: process.env.TWITTER_SECRET,
    version: "2.0",
    checks: "pkce",
    authorization: {
      params: {
        scope: "tweet.read users.read offline.access bookmark.read",
      },
    },
}),
```

### How to reproduce ☕️

1. Override the scope in the Twitter Provider
2. Create a callback to access the token
3. The object from the callback contains the default scope not the new one

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by ThangHuuVu** [comment]
Did you try the latest next-auth version? If possible, please add more information by providing a minimal reproduction repo and link it here, it would be super helpful to debug 🙏
